### ISSUE TITLE
Background blur in pop-up menus causes UI slow-down on Windows

### ISSUE THREAD

**Issue by nucspl** [crashmoney/obsidian-typewriter#27]
Since I began using a laptop with one fixed to 165hz, the settings panel was just really... slow? I tested out the default theme and that doesn't suffer from this issue, so I guess that it's isolated to Typewriter.

It's not that big of an issue in day-to-day use, but when having to go in and tweak something, you really feel it.

**Comment by crashmoney** [comment]
Hm, I haven't run into this issue before. There are a few questions that could help to debug this:

- Does this happen only on 165hz displays, or on other displays as well?
- Do you have any other snippets/plugins enabled that could be causing this behaviour?
- Does this lag happen when triggering the quick opener/command palette as well?
- Are Typewriter and Obsidian both on the latest version?

**Comment by nucspl** [comment]
- Not sure about other displays (I don't have a 60hz one around anymore to compare) but I've only noticed this once I made the switch.
- None of my plugins nor snippets are the cause.
- Just checked it now and it does, yes. It's most noticeable when scrolling, which I find odd since the file explorer/notes are just fine. I think it has something to do with the blur effect.
- And the theme is on 2.1.3, with my current Obsidian version being 1.0.3 (1.1.9 was really buggy), but the issue still persists in later versions.
